css/values.css

```diff
@@ -0,0 +1,51 @@
+.light-mode:root {
```

Comment:
I meant setting those variables directly in `:root` to make them even more global.
```suggestion
:root {
```

Reply:
Those are scoped. Removing those classes means the styles are not applied to the right themes.

Reply:
Just to reiterate - root sets variables globally, and this just scopes it to have a class. I rearranged the order for clarity.

Reply:
So we would have to add the `light-mode` class to the `<html>` tag, as you initially suggested, since there may be users who disable JavaScript, and the page would look like this:
![image](https://github.com/user-attachments/assets/24b146aa-0744-4762-8979-e4fac0ae2493)

Reply:
Put the light-mode tag back in.